.github/workflows/maven-snapshot.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish package to the Maven Central Repository
+on:
+  push:
+    branches:
+      - staging
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.7
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Import GPG key
+        env:
+          GPG_SECRET_KEYS: ${{ secrets.GPG_SECRET_KEYS }}
+          GPG_OWNERTRUST: ${{ secrets.GPG_OWNERTRUST }}
+        run: |
+          echo $GPG_SECRET_KEYS | base64 --decode | gpg --import --no-tty --batch --yes
+          echo $GPG_OWNERTRUST | base64 --decode | gpg --import-ownertrust --no-tty --batch --yes
+          echo "GPG_KEY_ID=$(gpg -k --with-colons | grep 'pub' | cut -d':' -f5 | tail -c 9)" >> $GITHUB_ENV
+      - name: Publish to maven central staging repository
+        run: make ci-publish-staging
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
You might have to change this `secrets.SONATYPE_USERNAME` to `secrets. OSSRH_USERNAME` and 
`secrets.SONATYPE_PASSWORD` to `secrets. OSSRH_PASSWORD`